pb_plugins/dcsdkgen/struct.py

```diff
@@ -5,42 +5,40 @@
                     is_request,
                     is_response,
                     is_struct)
-from .name import NameParser
+from .name_parser import NameParser
 from jinja2.exceptions import TemplateNotFound
 
 
 class Struct(object):
     """ Struct """
 
-    def __init__(self, package, template_env, pb_struct):
-        try:
-            self._template = template_env.get_template("struct.j2")
-        except TemplateNotFound:
-            # We don't always generate code related to structs (but still
-            # collect them)
-            pass
-
+    def __init__(self, plugin_name, package, template_env, pb_struct):
+        self._plugin_name = NameParser(plugin_name)
+        self._package = NameParser(package)
+        self._template = template_env.get_template("struct.j2")
         self._name = NameParser(pb_struct.name)
         self._fields = []
-        self._rpc_type = (package.title().replace(".", "_")
-                          + "_"
-                          + self._name)
 
         for field in pb_struct.field:
-            self._fields.append((field.json_name, extract_string_type(field)))
+            self._fields.append(
+                (NameParser(
```

Comment:
Please keep the parameter of `NameParser` in the same line, I had to look over it three times to get it!

Reply:
autopep8 is doing that :-/. I will check if there is a way to configure it.